src/frontend/components/UI/Sidebar/index.scss

```diff
@@ -42,6 +51,7 @@
   cursor: pointer;
   font-size: var(--text-md);
   color: var(--navbar-accent);
+  color: var(--navbar-inactive, var(--navbar-accent));
```

Comment:
this color is overriding the previous declaration, I see this repeated pattern of using 2 colors in other parts of this file too which looks incorrect

also, related to this change, themes need to be updated to use these new variables, for example this is how Marine's sidebar look now:
![image](https://user-images.githubusercontent.com/188464/215351787-c3f32c2a-5523-4ea0-946b-3860b94caa19.png)

All themes now have the same active/inactive colors of the default theme instead of the actual theme's accent

Reply:
Fixed it, I did not add all variables like you said, I added the default theme on the Themes.css, removed it from colors.css. Also fixed a few variables to match the correct style.

Reply:
![image](https://user-images.githubusercontent.com/26871415/216041599-9734abdd-38f1-41b7-a73d-175db219fdc7.png)